webpack/test-coverage.config.js

```diff
@@ -0,0 +1,12 @@
+import _ from 'lodash';
+import { jsLoader } from './base.config';
+import testConfig from './test.config';
+
+export default _.extend({}, testConfig, {
+  module: {
+    loaders: [
+      { test: /\.js/, loader: `${jsLoader}!isparta`, exclude: /node_modules|test[\/\\]/ },
+      { test: /\.js/, loader: `${jsLoader}`, include: /test[\/\\]/ }
+    ]
```

Comment:
Note the `isparta` loader to the non-test files. Also this is using the testConfig as it's base.